gascalc/4-paymaster-postop.gas.ts

```diff
@@ -0,0 +1,45 @@
+import { parseEther } from 'ethers/lib/utils'
+import { TestPaymasterAcceptAll__factory } from '../typechain'
+import { ethers } from 'hardhat'
+import { GasChecker } from './GasChecker'
+import { Create2Factory } from '../src/Create2Factory'
+import { hexValue } from '@ethersproject/bytes'
+
+const ethersSigner = ethers.provider.getSigner()
+
+context('Paymaster with PostOp', function () {
```

Comment:
Can't this test be "parametrized" in `2-paymaster.gas.ts` file? How much of it is the same?

Reply:
Yes it seems very "parametizable"